### PR TITLE
More google fixes

### PR DESCRIPTION
**Pull request by ruudboon**
Some more changes are needed to make google happy. Image is not in the assets repo. So I linked it just like the other blog images.

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -36,8 +36,9 @@ <h2 itemprop="name headline" class="headline">
                     {% if post.image %}
                     <img src="{{ page.image }}" alt="{{ page.title }}" itemprop="image">
                     {% else %}
-                    <meta itemprop="image" content="/assets/files/phalcon-logo-320x180.png">
+                    <meta itemprop="image" content="{{ site.url }}/assets/files/phalcon-logo-320x180.png">
```
Comment (niden) [inline _layouts/post.html]: `{{ site.app.url }}` <- more accurate because we set it.

Comment (niden) [reply _layouts/post.html]: Also check if these files exist in the assets repository (under `phalcon` folder). Then we can cross link them there.

Comment (ruudboon) [reply _layouts/post.html]: There not in the assets available. Already checked. Can add them there if you prefer.

**Comment by niden** [comment]
@Arhell can you check this PR and the logo files that it references so that we can add them to the `assets` repo please?

**Comment by niden** [comment]
Thank you @ruudboon

**Pull request merged**